services/galley/src/Galley/API/Teams.hs

```diff
@@ -694,6 +699,16 @@ ensureNotTooLarge tid = do
     throwM tooManyTeamMembers
   return $ TeamSize size
 
+-- FUTUREWORK: Large teams cannot have legalhold enabled
```

Comment:
Is the FUTUREWORK to remove this limitation?

Reply:
yes, added a better comment